### ISSUE TITLE
Can't manage or edit existing app after docker image update

### ISSUE THREAD

**Issue by maui1911** [Budibase/budibase#12281]
**Checklist**
- [x] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**
- Self
  - Method: docker-compose
  - Budibase Version: 2.12.7
  - App Version: <version> 

**Describe the bug**
After a docker image update? (i saw it restarted it only does that when watchtower updates the images)

Clicking the app from APPS gives:
![image](https://github.com/Budibase/budibase/assets/1742072/d4fa0a1f-3776-4068-b757-ac751f7fb88c)

error from diagnostics (probably):
{"level":"ERROR","timestamp":"2023-11-03T20:08:36.799Z","service":"@budibase/server","err":{"type":"Error","message":"Inaccessible host: `localhost' at port `9000'. This service may not be available in the `us-east-1' region.","stack":"UnknownEndpoint: Inaccessible host: `localhost' at port `9000'. This service may not be available in the `us-east-1' region.\n    at Request.ENOTFOUND_ERROR (/node_modules/aws-sdk/lib/event_listeners.js:529:46)\n    at Request.callListeners (/node_modules/aws-sdk/lib/sequential_executor.js:106:20)\n    at Request.emit (/node_modules/aws-sdk/lib/sequential_executor.js:78:10)\n    at Request.emit (/node_modules/aws-sdk/lib/request.js:686:14)\n    at error (/node_modules/aws-sdk/lib/event_listeners.js:361:22)\n    at ClientRequest.<anonymous> (/node_modules/aws-sdk/lib/http/node.js:99:9)\n    at ClientRequest.emit (node:events:517:28)\n    at ClientRequest.emit (node:domain:489:12)\n    at Socket.socketErrorListener (node:_http_client:501:9)\n    at Socket.emit (node:events:517:28)\n    at Socket.emit (node:domain:489:12)\n    at emitErrorNT (node:internal/streams/destroy:151:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:116:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)","code":"UnknownEndpoint","region":"us-east-1","hostname":"localhost","retryable":true,"originalError":{"type":"Error","message":"connect ECONNREFUSED 127.0.0.1:9000","stack":"Error: connect ECONNREFUSED 127.0.0.1:9000\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)\n    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)","errno":-111,"code":"NetworkingError","syscall":"connect","address":"127.0.0.1","port":9000,"region":"us-east-1","hostname":"localhost","retryable":true,"time":"2023-11-03T20:08:36.798Z"},"time":"2023-11-03T20:08:36.798Z"},"pid":210,"tenantId":"default","appId":"app_dev_1118fdb843984f49a3abb4bc7a921ba6","identityId":"us_8fc900313998485fb5a16e1ec5019625","identityType":"user","correlationId":"52c6a663-0a28-4042-91b8-9e4ea6312fc6","msg":"component-manifest-objectstore=failed appId=app_dev_1118fdb843984f49a3abb4bc7a921ba6 path=app_dev_1118fdb843984f49a3abb4bc7a921ba6/manifest.json

**To Reproduce**
1. Go to 'Apps'
2. Click on an APP
3. See error

**Expected behavior**
Not an error

**Docker-compose.yaml**
```
version: "3"

services:
  budibase:
    container_name: budibase
    restart: unless-stopped
    image: budibase/budibase:latest
    ports:
      - "10000:80"
    environment:
      JWT_SECRET: 
      MINIO_ACCESS_KEY: 
      MINIO_SECRET_KEY: 
      REDIS_PASSWORD: 
      COUCHDB_USER: 
      COUCHDB_PASSWORD: 
      INTERNAL_API_KEY: 
    volumes:
      - budi_data:/data
    depends_on:
      - db
  db:
    container_name: database
    image: postgres:15.4
    restart: unless-stopped
    environment:
      - POSTGRES_USER=
      - POSTGRES_PASSWORD=
    ports:
      - '5432:5432'
    volumes: 
      - db_data:/var/lib/postgresql/data
```
**Desktop (please complete the following information):**
 - OS: windows
 - Browser: firefox
 - Version: win 11

**Additional context**

**Comment by Hertizch** [comment]
Same issue here

> {"level":"ERROR","timestamp":"2023-11-04T05:17:22.233Z","service":"@budibase/server","err":{"type":"Error","message":"Inaccessible host: `localhost' at port `9000'. This service may not be available in the `us-east-1' region.","stack":"UnknownEndpoint: Inaccessible host: `localhost' at port `9000'. This service may not be available in the `us-east-1' region.\n    at Request.ENOTFOUND_ERROR (/node_modules/aws-sdk/lib/event_listeners.js:529:46)\n    at Request.callListeners (/node_modules/aws-sdk/lib/sequential_executor.js:106:20)\n    at Request.emit (/node_modules/aws-sdk/lib/sequential_executor.js:78:10)\n    at Request.emit (/node_modules/aws-sdk/lib/request.js:686:14)\n    at error (/node_modules/aws-sdk/lib/event_listeners.js:361:22)\n    at ClientRequest.<anonymous> (/node_modules/aws-sdk/lib/http/node.js:99:9)\n    at ClientRequest.emit (node:events:517:28)\n    at ClientRequest.emit (node:domain:489:12)\n    at Socket.socketErrorListener (node:_http_client:501:9)\n    at Socket.emit (node:events:517:28)\n    at Socket.emit (node:domain:489:12)\n    at emitErrorNT (node:internal/streams/destroy:151:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:116:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)","code":"UnknownEndpoint","region":"us-east-1","hostname":"localhost","retryable":true,"originalError":{"type":"Error","message":"connect ECONNREFUSED 127.0.0.1:9000","stack":"Error: connect ECONNREFUSED 127.0.0.1:9000\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)\n    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)","errno":-111,"code":"NetworkingError","syscall":"connect","address":"127.0.0.1","port":9000,"region":"us-east-1","hostname":"localhost","retryable":true,"time":"2023-11-04T05:17:22.233Z"},"time":"2023-11-04T05:17:22.233Z"},"pid":220,"tenantId":"default","appId":"app_dev_56978481c39140349538b7934b0a92e7","identityId":"us_e47d311edb944ab29ef4255f99d76d7a","identityType":"user","correlationId":"11bf3978-49be-4ddd-9cc5-3c07bf3de468","msg":"component-manifest-objectstore=failed appId=app_dev_56978481c39140349538b7934b0a92e7 path=app_dev_56978481c39140349538b7934b0a92e7/manifest.json"}
{"level":"ERROR","timestamp":"2023-11-04T05:17:22.377Z","service":"@budibase/server","err":{"type":"Error","message":"Inaccessible host: `localhost' at port `9000'. This service may not be available in the `us-east-1' region.","stack":"UnknownEndpoint: Inaccessible host: `localhost' at port `9000'. This service may not be available in the `us-east-1' region.\n    at Request.ENOTFOUND_ERROR (/node_modules/aws-sdk/lib/event_listeners.js:529:46)\n    at Request.callListeners (/node_modules/aws-sdk/lib/sequential_executor.js:106:20)\n    at Request.emit (/node_modules/aws-sdk/lib/sequential_executor.js:78:10)\n    at Request.emit (/node_modules/aws-sdk/lib/request.js:686:14)\n    at error (/node_modules/aws-sdk/lib/event_listeners.js:361:22)\n    at ClientRequest.<anonymous> (/node_modules/aws-sdk/lib/http/node.js:99:9)\n    at ClientRequest.emit (node:events:517:28)\n    at ClientRequest.emit (node:domain:489:12)\n    at Socket.socketErrorListener (node:_http_client:501:9)\n    at Socket.emit (node:events:517:28)\n    at Socket.emit (node:domain:489:12)\n    at emitErrorNT (node:internal/streams/destroy:151:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:116:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)","code":"UnknownEndpoint","region":"us-east-1","hostname":"localhost","retryable":true,"originalError":{"type":"Error","message":"connect ECONNREFUSED 127.0.0.1:9000","stack":"Error: connect ECONNREFUSED 127.0.0.1:9000\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)\n    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)","errno":-111,"code":"NetworkingError","syscall":"connect","address":"127.0.0.1","port":9000,"region":"us-east-1","hostname":"localhost","retryable":true,"time":"2023-11-04T05:17:22.377Z"},"time":"2023-11-04T05:17:22.377Z"},"pid":220,"tenantId":"default","appId":"app_dev_56978481c39140349538b7934b0a92e7","identityId":"us_e47d311edb944ab29ef4255f99d76d7a","identityType":"user","correlationId":"11bf3978-49be-4ddd-9cc5-3c07bf3de468","msg":"component-definitions=failed"}

**Comment by JezzBott** [comment]
I have the same errors after updating docker version 2.12.6.
Every time I click on any of the Apps I get the 'Error initialising app:'
![CleanShot 2023-11-04 at 10 36 34](https://github.com/Budibase/budibase/assets/19362393/57bea215-6ca0-4347-8194-37b5946bac06)

**Comment by mike12345567** [comment]
This issue should be resolved once the release of 2.12.11 is finished (within the next hour).

**Comment by maui1911** [comment]
thanks for the quick fix!
